src/test/scala/fr/polytechnique/cmap/cnam/SharedContext.scala

```diff
@@ -22,7 +22,7 @@ abstract class SharedContext
   Locale.setDefault(Locale.US)
   TimeZone.setDefault(TimeZone.getTimeZone("UTC"))
 
-  protected val debug: Boolean = false
+  protected val debug: Boolean = true
```

Comment:
This should always remain false.

Reply:
Yes my bad. Forgot to reset it to false.